### DISCUSSION TITLE
Solid color wallpapers menu redesign

### DISCUSSION THREAD

**Discussion by Qronikarz** [linuxmint/discussions#702] [Ideas]
Edit: Repurposing discussion for a better wallpaper UI since the solid color is already implemented but not presented well enough.

Old topic content:
> Currently to set your desktop wallpaper to a solid color you need to:
> 1. open some image editor
> 2. Set your screen size
> 3. Fill it with chosen color
> 4. save the picture
> 5. right click the picture file and set it as a wallpaper.
> 
> There are many included wallpapers by default, but there's no option to just choose one solid color.
> 
> I also found something like that - https://www.tc3.dev/posts/2021-09-04-gnome-3-solid-color-background/ so maybe it's possible to implement it without too many problems.
> 
> Here's how it looks in Windows (image credit: https://www.tc3.dev/posts/2021-09-04-gnome-3-solid-color-background/):
> ![Desktop_Background_Color-1](https://github.com/user-attachments/assets/fd806706-5b47-491a-bd30-dda9ffb04c75)

New:
As was pointed the option to set a solid color wallpaper already exists, but in my opinion it's not presented well.
Looking at above example it's a little better.

![Zrzut ekranu z 2024-12-10 23-15-56](https://github.com/user-attachments/assets/49d1b240-d2ed-4e36-b3ad-3b15ec72adee)
![Zrzut ekranu z 2024-12-10 23-16-01](https://github.com/user-attachments/assets/ad6d91f4-a42e-41b7-8995-3b35595008f5)

That's how it looks right now and few opinions about it:
1. There's no need to have 2 tabs - pictures and settings. Those 3 (or 5 when you enable the slideshow) can fit in the pictures tab.
2.  Picture format option should be presented before the pictures. This way if somebody chooses the "no image" the pictures can be hidden
3. "solid color" feels more appropriate than "no image". The latter is technically correct, but the first tell that the user can pick a custom color.
4.  Alternatively solid color option can be moved to the catalog menu on the side. There can be few predetermined colors and an input to set your own RGB Hexcode.

**Comment by Secret-chest** [comment]
In the options tab, you can choose the image display mode in a dropdown, and you can choose to have `No image`.

**Reply by Qronikarz** [reply]
You're right. In this case I think this whole window needs a redesign because it's not really well discoverable.
